packages/expect/src/utils.ts

```diff
@@ -319,6 +319,30 @@ export const typeEquality = (a: any, b: any): boolean | undefined => {
   return false;
 };
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const arrayBufferEquality = (a: any, b: any): boolean | undefined => {
```

Comment:
```suggestion
export const arrayBufferEquality = (a: unknown, b: unknown): boolean | undefined => {
```

`instanceof` check should be enough for this to work

Reply:
Thanks